mods/ctf_pvp_engine/ctf_flag/flag_func.lua

```diff
@@ -210,6 +210,14 @@ ctf_flag = {
 					minetest.chat_send_player(name, "You are at peace with this team!")
 					return
 				end
+				
+				local glitcher = players_glitching[name]
+				if glitcher then
+					local player = minetest.get_player_by_name(name)
+					player:setpos(glitcher)
+					minetest.chat_send_player(name, "You can not capture a flag while in airplane mode!")
```

Comment:
Could do with a more generic message like: `"You can't capture the flag by glitching!"`